src/D2L.CodeStyle.Analyzers/Async/Generator/SourceFilePathHintNameGenerator.cs

```diff
@@ -0,0 +1,110 @@
+﻿#nullable enable

```

Comment:
Oh, I should note I'm manually enabling this in all my files.

I did a quick look at enabling this for the whole project and it was a little daunting. It was one of the contributor to my other questions on Slack about deleting particular analyzers :P

Not impossible to migrate the whole thing, I just haven't gotten around to it.

Reply:
Can do do the opposite.  Disable all existing files and enabled by default?

Reply:
I'm fine with that in general, just wasn't bothering with in this PR anyway.

Reply:
Good to remove now

Reply:
Done